exposed-jodatime/src/test/kotlin/org/jetbrains/exposed/JodaTimeTests.kt

```diff
@@ -1,16 +1,53 @@
 package org.jetbrains.exposed
 
+import org.jetbrains.exposed.sql.SchemaUtils
+import org.jetbrains.exposed.sql.Table
+import org.jetbrains.exposed.sql.insert
+import org.jetbrains.exposed.sql.jodatime.*
+import org.jetbrains.exposed.sql.select
 import org.jetbrains.exposed.sql.tests.DatabaseTestsBase
+import org.jetbrains.exposed.sql.tests.TestDB
 import org.jetbrains.exposed.sql.tests.currentDialectTest
+import org.jetbrains.exposed.sql.tests.shared.assertEquals
 import org.jetbrains.exposed.sql.vendors.MysqlDialect
 import org.joda.time.DateTime
 import org.joda.time.DateTimeZone
+import org.joda.time.LocalDateTime
+import org.junit.Test
 import kotlin.test.assertEquals
 
 open class JodaTimeBaseTest : DatabaseTestsBase() {
     init {
         DateTimeZone.setDefault(DateTimeZone.UTC)
     }
+
+    @Test
+    fun jodaTimeFunctions() {
+        withTables(listOf(TestDB.SQLITE), Cities) {
```

Comment:
I added listOf(TestDB.SQLITE) to exclude sqlite database from the test because strftime('%Y', dateColumn) returns null. When i did the debugging without calling a time function i found that only sqlite is storing the datetime as integer format. I think that the datetime is not well formatted in sqlite dialect.

Reply:
The problem here is that Exposed stores datetime as unixtime with milliseconds while by default SQLite expect unixtime with seconds only.
You could workaround it by using ` / 1000, 'unixepoch')` construnction like: 
```kotlin
        append("STRFTIME('%Y',")
        append(expr)
        append(" / 1000, 'unixepoch')")
```

Reply:
@Tapac that works and returns a correct value but raises an exception because it expects a number but it gets a string value. There are two solutions I tested : 
- adding cast : 
        append("CAST(")
        append("STRFTIME('%M',")
        append(expr)
        append(" / 1000, 'unixepoch')")
        append("AS INTEGER)")

OR :  adding string expectation : 

class IntegerColumnType : ColumnType() {
......
    override fun valueFromDB(value: Any): Any = when(value) {
        is Int -> value
....
        **is String -> value.toInt()**
        else -> error("Unexpected value of type Int: $value of ${value::class.qualifiedName}")
    }
}

Reply:
I guess the extending `IntegerColumnType` is fine